src/avram/model.cr

```diff
@@ -257,6 +225,21 @@ abstract class Avram::Model
       {% value = nil %}
     {% end %}
 
+    @[DB::Field(
+      ignore: false,
+      key: {{ type_declaration.var }},
+      {% if is_serialized %}
+      converter: JSONConverter({{ data_type }}),
+      {% elsif data_type.id == Array(Float64).id %}
+      converter: PG::NumericArrayFloatConverter,
```

Comment:
I'm not sure why this has to be converted now when it didn't need to be with `DB.mapping` :man_shrugging:

Reply:
It looks like this logic was inside the `setup_db_mapping` macro before